pkg/types/nutanix/helpers.go

```diff
@@ -27,24 +27,37 @@ type metadataCloudInit struct {
 	UUID string `json:"uuid"`
 }
 
-func bootISOImageName(infraID string) string {
+// BootISOImageName is the image name for Bootstrap node for a given infraID
+func BootISOImageName(infraID string) string {
 	return fmt.Sprintf("%s-%s", infraID, isoFile)
 }
 
-func createBootstrapISO(infraID, userData string) (string, error) {
+// BootISOImagePath is the image path for Bootstrap node for a given infraID and path
+func BootISOImagePath(path, infraID string) string {
+	imgName := BootISOImageName(infraID)
+	application := "openshift-installer"
+	subdir := "image_cache"
+	fullISOFile := fmt.Sprintf("%s/%s/%s/%s", path, application, subdir, imgName)
```

Comment:
The path should be created using [filepath.Join](https://pkg.go.dev/path/filepath#Join) to accommodate OSes with different path separators.

Reply:
Good point. Let's use file path.